src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -957,21 +959,20 @@ public static EntityPlayer getCraftingPlayer()
     {
         return craftingPlayer.get();
     }
-    public static ItemStack getContainerItem(ItemStack stack)
+    @Nonnull
+    public static ItemStack getContainerItem(@Nonnull ItemStack stack)
     {
-        if (stack == null) return null;
-
         if (stack.getItem().hasContainerItem(stack))
```

Comment:
Isn't it possible to have an invalid stack that has an item which has a container item as the input here?